modules/clients/src/test/java/org/apache/ignite/jdbc/thin/JdbcThinPartitionAwarenessReconnectionAndFailoverSelfTest.java

```diff
@@ -244,10 +299,10 @@ public void testReconnectionDelayIncreasing() throws Exception {
             assertEquals("Unexpected log records count.", 4, logHnd.records.size());
 
             String expRecordMsg = "Failed to connect to Ignite node " +
-                "[url=jdbc:ignite:thin://127.0.0.1:10800,127.0.0.1:10810]. address = [localhost/127.0.0.1:10810].";
+                "[url=jdbc:ignite:thin://127.0.0.1:10800,127.0.0.1:10810]. address = [";
```

Comment:
Depending on OS and java version the local hostname may remain unresolved (in my case it prints `/127.0.0.1:10810` instead of `localhost/127.0.0.1:10810`).